CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## 16.0.0 (unreleased)
 
+- Updated simple.md @MdSahil-oss
```

Comment:
This is not under the correct subheading. Please move under this version's Documentation subheading.